infra/build/functions/base_images.py

```diff
@@ -109,49 +95,30 @@ def _get_introspector_base_images_steps(images, tag_prefix=TAG_PREFIX):
   return steps
 
 
-def get_logs_url(build_id, project_id='oss-fuzz-base'):
-  """Returns url that displays the build logs."""
-  return ('https://console.developers.google.com/logs/viewer?'
-          f'resource=build%2Fbuild_id%2F{build_id}&project={project_id}')
-
-
 # pylint: disable=no-member
-def run_build(steps, images, build_version=MAJOR_TAG):
-  """Execute the retrieved build steps in gcp."""
-  credentials, _ = google.auth.default()
-  build_body = {
-      'steps': steps,
-      'timeout': str(6 * 3600) + 's',
-      'options': {
-          'machineType': 'N1_HIGHCPU_32'
-      },
+def run_build(steps, images, tags=None, build_version=MAJOR_TAG):
+  """Execute the retrieved build steps in gcb."""
+  credentials = oauth2client.client.GoogleCredentials.get_application_default()
```

Comment:
Could you explain why this check is needed?

Reply:
Oops yes, I did mean "change".

oauth2client is deprecated per https://pypi.org/project/oauth2client/. We shouldn't be using it. The recommended way to do this now is using `google.auth`